user_docs/en/userGuide.t2t

```diff
@@ -504,7 +504,7 @@ When the menu comes up, You can use the arrow keys to navigate the menu, and the
 | Stop speech | Control | control | 2-finger tap | Instantly stops speaking |
 | Pause Speech | shift | shift | none | Instantly pauses speech. Pressing it again will continue speaking where it left off (if pausing is supported by the current synthesizer) |
 | NVDA Menu | NVDA+n | NVDA+n | 2-finger double-tap | Pops up the NVDA menu to allow you to access preferences, tools, help, etc. |
-| Toggle Speech Mode | NVDA+s | NVDA+s | none | Toggles speech mode between speech, beeps and off. |
+| Toggle Speech Mode | NVDA+s | NVDA+s | none | Toggles speech mode between speech, speech on demand, beeps and off. |

```

Comment:
I think it would be worth adding a new user guide section "Speech mode" to explain each of these modes. This will also be useful when creating #15806

Reply:
@seanbudd can this be done in a follow-up?

Reply:
I agree that a more detailed description in the User Guide would be nice.

@seanbudd, @Qchristensen  or any other reviewer, where would you see this more detailed description in the User Guide? It's a bit tricky since speech mode is not a configuration parameter, i.e. it's not saved when restarting NVDA.

Reply:
> any other reviewer, where would you see this more detailed description in the User Guide? It's a bit tricky since speech mode is not a configuration parameter, i.e. it's not saved when restarting NVDA.


Either in Chapter `4. Using NVDA`, specifically as a new section `4.8. Speech Modes`; or
in chapter 12, subchapter `12.2. miscellaneous Settings`, specifically in a new section `12.2.4. Speech Modes`.